content/documentation/staging/tutorial/05-connect/01-request-routing.adoc

```diff
@@ -0,0 +1,158 @@
+    <img src="/images/tutorial/05-01-deployment-istio-config.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+In our example the link:https://github.com/kiali/demos/blob/master/travels/travels-v2.yaml#L15[HTTP Headers] are added as part of the trace context.
+
+Then tracing will populate custom tags with the *portal*, *device*, *user* and *travel* used.
+
+NOTE: *[3]* Use the Request Routing Wizard on *travels* service to generate a traffic rule
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/05-01-travels-request-routing.png" title="Travels Service Request Routing">
+    <img src="/images/tutorial/05-01-travels-request-routing.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+In the first rule, we will add a Request Matching for *portal* header equals to *travels.uk* value.
+
+Use "Add Match" button to update the "Matching selected" for this rule.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/05-01-add-match.png" title="Add Request Matching">
+    <img src="/images/tutorial/05-01-add-match.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+Move to "Route To" tab to update the destination for this "Request Matching".
+
+Use the "Add Rule" to create the first rule.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/05-01-route-to.png" title="Route To">
+    <img src="/images/tutorial/05-01-route-to.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+Add similar rules to route traffic from *viaggi.it* to *travels-v2* workload and from *voyages.fr* to *travels-v3* workload.
+
+When the three rules are defined you can use "Create" button to generate all Istio configuration needed for this scenario.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/05-01-rules-defined.png" title="Rules Defined">
+    <img src="/images/tutorial/05-01-rules-defined.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+The Istio Config linked with a given service can also be located under the "Istio" tab in the Service Details page.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/05-01-service-istio-config.png" title="Service Istio Config">
+    <img src="/images/tutorial/05-01-service-istio-config.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+NOTE: *[4]* Verify Request Routing working from the *travels-portal* Graph
+
+Once the Request Routing is working we can verify that outbound traffic from every portal goes all to a single *travels* workload.
+
+If you use the "Find.../Hide..." filter, graph can hide old traffic grey edges for a clear graph.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/05-01-request-routing-graph.png" title="Travels Portal Namespace Graph">
+    <img src="/images/tutorial/05-01-request-routing-graph.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
```

Comment:
This is no longer necessary as by default we no longer show "Idle Edges".  Only if you enable the "Show Idle Edges" display option is the `!traffic` Hide option useful.

Reply:
But I think it applies for the version 1.28.0 used for the tutorial.
We can change that later, once we consolidate it, we can review it when we publish new versions if some feature needs some adjustment.